### ISSUE TITLE
Build Errors in xaudio2.cpp on Windows?

### ISSUE THREAD

**Issue by kadiera** [espeak-ng/pcaudiolib#22]
Started working on a language file for Lakota. I'm trying to build it so that I can test how it sounds before submitting. I get the following errors, which, since I didn't edit these files, seems odd...

<img width="663" alt="image" src="https://user-images.githubusercontent.com/83197488/210191353-a8bf7070-0031-46f8-9399-55aba2bd7a4a.png">

Any thoughts on what is going on?

**Comment by helloclq** [comment]
Have you already solved this problwm? any update?

**Comment by kadiera** [comment]
I have not, though I assume this is a user error problem - my last real coding experience was 20 years ago using Fortran :)

I did have a file missing, but even fixing that issue did not resolve all the issues.

So, my theory at this point is to find someone who has some time and a little more experience with this stuff, get them to figure out the build issues, and then I can hear the speech it generates, and should be able to tweak my sound files and recompile.....

**Comment by lucasjinreal** [comment]
TLDR: this lib doesn't support or the maintainer didn't test on windows at all. Given it up.

**Comment by thewh1teagle** [comment]
Once this pr merged

- https://github.com/espeak-ng/pcaudiolib/pull/32

It will be easy to build on windows. CI build successfully.
